Fatima-justinNykiel.js

```diff
@@ -0,0 +1,109 @@
+function calculateCube(num) {
+    return Math.pow(num, 3);
+}
+console.log(calculateCube(5));
+
+function isAVowel(char) {
+    char = char.toLowerCase();
```

Comment:
nice work recognizing that parameters are mutable! makes your code really concise by not introducing an intermediate variable